Refactor routes

andygw opened the pull request:
So much cleaner...

![image](https://user-images.githubusercontent.com/48395382/202299741-865bd745-1e68-41bc-92ba-7bb8f004d786.png)

andygw commented:
Important note: seed account PWs changed to `111111`

squush commented:
Sounds good. Why the change?

Le mer. 16 nov. 2022 à 19:31, Andrew ***@***.***> a écrit :

> Important note: seed account PWs changed to 111111
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/squush/AirBnE/pull/42#issuecomment-1317873750>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AHT4ANTE2CEC3ABVBVRCRD3WIV4GVANCNFSM6AAAAAASCWDORY>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>

andygw commented:
Easier to type, I guess? I can change it back if you have the pws saved already

andygw commented:
Changed the seed to have some consistent users, crimes, and bookings each time. The My Bookings page will automatically look like this after each re-seed:

![image](https://user-images.githubusercontent.com/48395382/202327571-4be9f8fa-e648-44ac-b794-7ea5c2366f24.png)

There will also be some random users and crimes added after this, to seed the All Crimes page a bit.

Also added a couple to-do notes.

(This should have been on a different branch but I screwed up)

Sreg1 commented on app/views/bookings/_form.html.erb:
```diff
@@ -1,5 +1,6 @@
 <%= simple_form_for [@crime, @booking] do |f| %>
   <%= f.input :target %>
+  <%# change to calendar / look up 'flatpciker' %>
```
did you mean "flatpickr"
